fe-angular/Jenkinsfile

```
@@ -14,7 +14,7 @@ node {
 
 library("ods-jenkins-shared-library@${sharedLibraryRef}")
 
-def angularCliVersion = "13.3.0"
+def angularCliVersion = "15.2.4"
```

Latest version has changed a few hours ago (https://www.npmjs.com/package/@angular/cli). How about
```suggestion
def angularCliVersion = "15.2.7"
```

---

The test were successfull so version updated :)